src/ontology/bridge/uberon-bridge-to-mba.obo

```diff
@@ -1391,45 +1402,45 @@ property_value: status "Verified" xsd:string {source="https://ror.org/03cpe7c52"
 
 [Term]
 id: MBA:12993
-relationship: BFO:0000050 NCBITaxon:10090
-relationship: BFO:0000050 UBERON:0008930
+intersection_of: BFO:0000050 NCBITaxon:10090 {all_some="true"}
```

Comment:
I don’t think the `all_some` qualifier is needed here. It’s not really explicitly said in the OBO Flat File format spec, but `all_some` seems to be the default (that’s how the OWL API treats it anyway). Again, it doesn’t do any harm, but since it’s not needed, probably better avoided?

Reply:
yeah notice that, but i figured it doesn't harm - mostly focused on the owl format - the obo is just i conversion using ROBOT